bundles/org.openhab.binding.meteofrance/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -39,6 +47,10 @@
 		<label>Storm</label>
 		<description>Storm alert level</description>
 		<category>oh:meteofrance:orage</category>
+		<tags>
+			<tag>Alarm</tag>
+			<tag>Level</tag>
```

Comment:
Rain or Wind would be probably more appropriate

Reply:
Changing to Alarm.Wind